code/modules/spells/spell_types/aimed.dm

```diff
@@ -54,6 +55,11 @@
 	var/obj/item/projectile/P = new projectile_type(user.loc)
 	P.current = get_turf(user)
 	P.preparePixelProjectile(target, get_turf(target), user)
+	if(projectile_damage_override != (-1))
```

Comment:
you don't need the brackets, when there's no first argument to ```-``` it's treated as a unary argument (basically produces something like ```0 - number```, since that'll flip the sign (since basic maths tells us A -- B is equivalent to A + B))